docs/installation/local.mdx

```diff
@@ -4,15 +4,40 @@ sidebar_position: 1
 
 # Local Installation
 
-Use Capact CLI to create local Kubernetes cluster and install Capact on it.
+Use Capact CLI to create a local Kubernetes cluster and install Capact on it.
 
 ## Prerequisites
+
 - [Docker](https://www.docker.com/) installed
 - Capact CLI installed according to the [Getting Started](./../cli/getting-started.mdx#install) guide
 
+### CPU and memory requirements
+
+Capact consists of multiple components, which you can find in [`deploy/kubernetes/charts`](https://github.com/capactio/capact/tree/main/deploy/kubernetes/charts). Make sure that Docker has enough resources to install Capact properly.
+
+#### Minimal:
+- 2CPUs
+- 4GB
+
+#### Recommended:
+- 4CPUs
+- 8GB
```

Comment:
As I suggested earlier, consider such a minor change:

(especially it would be good to clarify that the gigabytes refer to RAM)

```suggestion
#### Minimal:
- 2 CPU cores
- 4GB RAM

#### Recommended:
- 4 CPU cores
- 8GB RAM
```